Data/Sector/Hong-Kong-Sector-File.ese

```diff
@@ -1174,8 +1192,10 @@ DISPLAY:VHHK·VHHK_SCT_TME_01·000·250:VHHK·VHHK_SCT_TME_01·000·250:VHHK·VH
 DISPLAY:VHHK·VHHK_SCT_TME_01·000·250:VHHK·VHHK_SCT_TME_01·000·250:VHHK·VHHK_SCT_25_DEP_05·135·250
 DISPLAY:VHHK·VHHK_SCT_TRW_01·250·365:VHHK·VHHK_SCT_TRW_01·250·365:VHHK·VHHK_SCT_TRE_01·250·365
 DISPLAY:VHHK·VHHK_SCT_TRE_01·250·365:VHHK·VHHK_SCT_TRW_01·250·365:VHHK·VHHK_SCT_TRE_01·250·365
+DISPLAY:VHHK·VHHK_SCT_07NAP_DEP_03·000·250:VHHK·VHHK_SCT_07NAP_DEP_03·000·250:VHHK·VHHK_SCT_TME_01·000·250
+DISPLAY:VHHK·VHHK_SCT_TME_01·000·250:VHHK·VHHK_SCT_07NAP_DEP_03·000·250:VHHK·VHHK_SCT_07NAP_DEP_03·000·250
```

Comment:
Sectorline 32 and 35 (below) not displayed/missing in NAP configuration (syntax error)?

Example: Logged on as HKG_E_CTR, own TME, NAP 07 Configuration, VHHH_APP online
![image](https://github.com/vatsimhk/Hong-Kong-Sector-Package/assets/55276992/e5d80ff2-7481-4df2-a8a3-07a651b181ee)
```suggestion
DISPLAY:VHHK·VHHK_SCT_TME_01·000·250:VHHK·VHHK_SCT_TME_01·000·250:VHHK·VHHK_SCT_07NAP_DEP_03·000·250
```